bundles/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/handler/NetworkHandler.java

```diff
@@ -116,7 +119,7 @@ public void handleCommand(ChannelUID channelUID, Command command) {
     @Override
     public void partialDetectionResult(PresenceDetectionValue value) {
         updateState(CHANNEL_ONLINE, OnOffType.ON);
-        updateState(CHANNEL_LATENCY, new DecimalType(value.getLowestLatency()));
+        updateState(CHANNEL_LATENCY, new QuantityType<>(value.getLowestLatency(), SmartHomeUnits.MILLISECOND));
```

Comment:
Same question like in https://github.com/openhab/openhab-core/pull/798#issuecomment-489577501: 
I am not sure if we really need it. Doesn't `MILLI(SECOND)` do the job at all?

Reply:
Yes, I cancelled PR in OH core and will do this.